featuretools/tests/entityset_tests/test_es.py

```diff
@@ -934,3 +934,23 @@ def test_sizeof(entityset):
         total_size += entity.last_time_index.__sizeof__()
 
     assert entityset.__sizeof__() == total_size
+
+
+def test_construct_without_id():
+    assert ft.EntitySet().id is None
+
+
+def test_repr_without_id():
+    match = 'Entityset: None\n  Entities:\n  Relationships:\n    No relationships'
+    assert repr(ft.EntitySet()) == match
+
+
+def test_getitem_without_id():
+    error_text = 'Entity test does not exist in None'
+    with pytest.raises(KeyError, match=error_text):
+        ft.EntitySet()['test']
+
+
+def test_metadata_without_id():
+    es = ft.EntitySet()
+    assert es.create_metadata_dict().get('id') is None
```

Comment:
the metadata dict is also used when serializing and loading an entityset. can we add one test that serializes and loads an entityset without an id? 

something like

```python
def test_to_pickle_id_none(entityset):
    entityset = ft.EntitySet()
    dirname = os.path.dirname(integration_data.__file__)
    path = os.path.join(dirname, 'test_entityset.p')
    if os.path.exists(path):
        shutil.rmtree(path)
    entityset.to_pickle(path)
    new_es = ft.read_pickle(path)
    assert entityset.__eq__(new_es, deep=True)
    shutil.rmtree(path)
```

Reply:
Yes not a problem. I made the adjustments described above. Let me know if anything else needs modification. Thanks.